lib/osf-components/addon/components/validated-input/date/template.hbs

```diff
@@ -16,5 +16,6 @@
         @minDate={{@minDate}}
         @maxDate={{@maxDate}}
         @disabled={{@disabled}}
+        @date={{''}}
```

Comment:
Do the quotes need to be wrapped in the curly braces? That seems like an odd construction.

Reply:
You're right. We should use `null`. I read the docs and figured that an empty string would probably be okay. I have updated the `ValidatedInput:Date` component to pass along the `date` param to `ember-flatpickr`.